enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/FileUserRealm.java

```diff
@@ -260,6 +268,31 @@ boolean deleteUser( String username ) throws IOException
         return result;
     }
 
+    void suspendUser( String username ) throws IOException, ConcurrentModificationException
```

Comment:
This method isn't properly synchronized and `ConcurrentModificationException` shouldn't be bubbled up. (See methods above)